priv/repo/migrations/20170317153551_create_message.exs

```diff
@@ -0,0 +1,15 @@
+defmodule Healthlocker.Repo.Migrations.CreateMessage do
+  use Ecto.Migration
+
+  def change do
+    create table(:messages) do
+      add :body, :string, null: false
+      add :name, :string
+      add :user_id, references(:users, on_delete: :nothing), null: false
+
```

Comment:
right now it's a general room so there's no to or from a particular user (and no private rooms). But I think that would be a good way to scope them when we have that as well. Do you reckon I should add messages scoped to a Room in this PR?